kernel/syscall.c

```diff
@@ -282,7 +282,13 @@ ssize_t sys_write(int fd, const char* buf, size_t len)
 	}
 
 	if (fd > 2)
-		i = socket_recv(s, &i, sizeof(i));
+	{
```

Comment:
Your if-braces do not quite match the rest of the file

Reply:
In this function alone, there are two different styles.
Which is the preferred one?

Reply:
Oh true. Nevermind. Sorry for bothering